src/data/url.js

```diff
@@ -0,0 +1,18 @@
+import { push } from 'svelte-spa-router'
+import qs from 'qs'
+
+export function updateQueryString(location, queryString, updates) {
+  let queryStringData = qs.parse(queryString)
```

Comment:
since you're only changing properties, you may b able to use `const` here.

Reply:
Huh, good to know. You're right: it does work as a `const`.

Per our discussion, I think I'll leave it as a `let` because it seems to communicate more clearly the intent (that this contains data which will be changed).